ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```diff
@@ -1,25 +1,28 @@
-tag_specification:
-  name: '4.0'
-  namespace: ocp
```

Comment:
@enxebre don't be afraid of the changes, the existing config was just tool-processed so the ordering is different. The only thing that should really changed is that `master` config is now 4.2-based, plus there is the new `promotion` stanza.